fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/TypeRdfContext.java

```diff
@@ -58,71 +61,36 @@ public TypeRdfContext(final FedoraResource resource,
 
         //include rdf:type for primaryType, mixins, and their supertypes
         concatRdfTypes();
+        if (resource() instanceof NonRdfSource) {
+            // gather versionability info from the parent
+            if (resource().getNode().getParent().isNodeType("mix:versionable")) {
```

Comment:
See above: "It's not being checked there because it's not being stored there. As to why, I do not know. You would have to go investigate that code. I assumed it was for a reasonable reason, but after all, this is Fedora."